tests/Xendit/TransactionTest.php

```diff
@@ -77,18 +77,6 @@ public function testDetailIsGettable()
         $this->assertEquals($result['id'], $expectedResponse['id']);
     }
 
-    /**
-     * Get list of transactions test
-     * Should throw ApiException
-     *
-     * @return void
-     */
-    public function testListIsGettableThrowsException()
-    {
-        $this->expectException(\Xendit\Exceptions\ApiException::class);
-        Transaction::list();
-    }
-
```

Comment:
I removed this test even though it's not related to QR code endpoint because it always fail. sample job: https://github.com/xendit/xendit-php/actions/runs/3229489365/jobs/5286886895

after checking locally, turns out this test will actually send an actual requests to our API, and it will respond with an actual HTTP response:
![image](https://user-images.githubusercontent.com/463775/195183970-a0c8a0e3-4913-4400-af03-8cadfceccc0d.png)